### PR TITLE
Update doc links

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
$subject.

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]

File: websocket-ballerina/Package.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Module Overview
+## Package Overview
 
 This module provides an implementation for connecting and interacting with WebSocket endpoints. The module facilitates three types of network entry points as ‘Client’, ‘AsyncClient’, and ‘Listener’.
```
Comment (ThisaruGuruge) [inline websocket-ballerina/Package.md:3]:
```suggestion
This package provides an implementation for connecting and interacting with WebSocket endpoints. The package facilitates three types of network entry points as ‘Client’, ‘AsyncClient’, and ‘Listener’.
```

File: websocket-ballerina/Package.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ service class WsService {
...
-* [WebSocket Basic Example](https://ballerina.io/swan-lake/learn/by-example/websocket-basic-sample.html)
-* [WebSocket Chat Application](https://ballerina.io/swan-lake/learn/by-example/websocket-chat-application.html)
-* [WebSocket Proxy Server](https://ballerina.io/swan-lake/learn/by-example/websocket-proxy-server.html)
-* [Client Endpoint](https://ballerina.io/swan-lake/learn/by-example/websocket-client.html) 
-* [Async Client Endpoint](https://ballerina.io/swan-lake/learn/by-example/websocket-async-client.html)  
+* [Basic Server Example](https://ballerina.io/learn/by-example/websocket-basic-sample.html)
+* [Chat Application](https://ballerina.io/learn/by-example/websocket-chat-application.html)
+* [Proxy Server](https://ballerina.io/learn/by-example/websocket-proxy-server.html)
+* [Client](https://ballerina.io/learn/by-example/websocket-text-client.html) 
+* [Async Client Endpoint](https://ballerina.io/learn/by-example/websocket-async-text-client.html)
```
Comment (shafreenAnfar) [inline websocket-ballerina/Package.md:54]: Shall we say `Async Client` instead of `Async Client Endpoint` ?

File: websocket-ballerina/Package.md [hunk 3]
```diff
@@ -47,8 +47,8 @@ service class WsService {
 **onPong**: The received `pong` messages are dispatched to this remote method.
 
 For more information, see the following.
-* [WebSocket Basic Example](https://ballerina.io/swan-lake/learn/by-example/websocket-basic-sample.html)
-* [WebSocket Chat Application](https://ballerina.io/swan-lake/learn/by-example/websocket-chat-application.html)
-* [WebSocket Proxy Server](https://ballerina.io/swan-lake/learn/by-example/websocket-proxy-server.html)
-* [Client Endpoint](https://ballerina.io/swan-lake/learn/by-example/websocket-client.html) 
-* [Async Client Endpoint](https://ballerina.io/swan-lake/learn/by-example/websocket-async-client.html)  
+* [Basic Server Example](https://ballerina.io/learn/by-example/websocket-basic-sample.html)
```
Comment (shafreenAnfar) [inline websocket-ballerina/Package.md:50]: Let's just say `Server Example`

File: websocket-ballerina/Package.md [hunk 4]
```diff
@@ -47,8 +47,8 @@ service class WsService {
...
+* [Chat Application](https://ballerina.io/learn/by-example/websocket-chat-application.html)
+* [Proxy Server](https://ballerina.io/learn/by-example/websocket-proxy-server.html)
```
Comment (shafreenAnfar) [inline websocket-ballerina/Package.md:52]: I thought we are removing these two.

Comment (Bhashinee) [reply websocket-ballerina/Package.md]: Removed them.

**Pull request merged**